.pre-commit-config.yaml

```diff
@@ -63,7 +63,7 @@ repos:
           - "B101,B311"
 
   - repo: https://github.com/myint/docformatter
-    rev: v1.5.0-rc1
+    rev: v1.5.0
```

Comment:
don't update this -- it does automatically in separated PR. This could lead to unknown breaking issues

Reply:
This didn't run for me. They couldn't find the version anymore. Can you check this again?

Reply:
this is weird since the ci passes: https://results.pre-commit.ci/run/github/145693916/1661761711.K3cMMmeFQVW7ZH-dHxjMjA

Reply:
Then it is good right?

Reply:
i mean, it passes with `v1.5.0` -- in any case let's not block this pr for that